src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -94,7 +92,28 @@ public enum SearchType
             Normal,
 
             //Search by category using the "." character for example "list.re"
-            ByCategory
+            ByDotCategory,
+
+            //The SearchTerm contains at least one empty space
+            ByEmptySpace
+        }
+
+        /// <summary>
+        /// This enum will used to create different type of Wildcard queries using regular expressions
```

Comment:
be used to

Reply:
comment updated, see commit: https://github.com/DynamoDS/Dynamo/pull/15473/commits/40697ac1b0752b02a53c5ccb86b77a1a67747492